hw/ip/aes/doc/dv_plan/index.md

```diff
@@ -59,26 +59,30 @@ The `aes_base_vseq` virtual sequence is extended from `cip_base_vseq` and serves
 All test sequences are extended from `aes_base_vseq`.
 It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
 Some of the most commonly used tasks / functions are as follows:
-* aes_init:    Configure aes control and can via a knob be set to randomize register content before starting a test.
+* aes_init:    Initialize the AES module from the randomized environment variables in the config.
 * set_op:      Set AES operation to encrypt or decrypt.
-* write_key:   Write initial key to aes init key registers.
+* write_key:   Write initial key to AES init key registers.
 * add_data:    Add the next 128 block to the input registers.
-* read_output:  Poll the status reg for dataready bit and read the result from aes output regs
+* read_output:  Poll the status reg for dataready bit and read the result from AES output regs
+* clear_reg:  Based on the input this function clears data input, data output or key registers or any combination of these.
+* set_manual_trigger: Chooses between AES auto start and manual start.
+* trigger_start:  Set the start bit to trigger a new encryption/decryption.
 
 #### Functional coverage
 To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
-The following covergroups have been developed to prove that the test intent has been adequately met:
+The following cover groups have been developed to prove that the test intent has been adequately met:
 *---  Work in Progress ---
 
 ### Self-checking strategy
 #### Scoreboard
 The `aes_scoreboard` is primarily used for end to end checking.
-It creates the following analysis fifos to retrieve the data monitored by corresponding interface agent:
-* tl_a_chan_fifo, tl_d_chan_fifo: These 2 fifos provides transaction items at the end of address channel and data channel respectively
-
+It creates the following analysis FIFOs to retrieve the data monitored by corresponding interface agent:
+* tl_a_chan_fifo, tl_d_chan_fifo: These 2 FIFOs provides transaction items at the end of address channel and data channel respectively
+each FIFO is monitored and incoming transactions are stored.
+Whenever 128 new bits or the transaction is finished the sequence item handed to a reference model that will generate the expected response. at the same time the scoreboard is waiting for the result of the AES module to compute. Once complete the result is scored against the prediction made by the reference model. The scoreboard will compare every 128bit segment. The reference model is implemented to be either a C-implementation or an SSL-library selected on a random basis with the default distribution of 80% OpenSSL/BoringSSL  20% C-model.
```

Comment:
I think we discussed this already, please:
- put one sentence per line (in the source file), 
- start new sentences with a capital letter,
- finish sentences with a full stop.

Reply:
What exactly do you mean by "Whenever 128 new bits or the transaction is finished"?

Reply:
This Was written before our last meeting.
and at the time we had discussed checking per block. 
which after out last talk is only possible when using the C-model as reference.

I have updated the section